### ISSUE TITLE
TCPServer slow down when there's a client connected on RPi4/buster

### ISSUE THREAD

**Issue by Bencilari** [openframeworks/openFrameworks#6513]
I've made a post on the forum, but i'm reporting here too, because i think it's a bug.
With Raspberry Pi4/buster, when i create a TCPServer listening for incoming connections, when a client connect, the app slow down a lot (from 60FPS to ~20/25).
You can test this behaviour with the TCPServer example ([https://github.com/openframeworks/openFrameworks/tree/patch-release/examples/communication/networkTcpServerExample](url))
Just run it, and connect (even from the same machine, with nc, or maybe (not tested) with the companion example TCPClient). So it's not a network issue.
Same app in other linux distros runs fine.

**Comment by arturoc** [comment]
Are you compiling the application in release mode?

**Comment by Bencilari** [comment]
Yes, compiled with `make -j4`. Tested also explicitly with `make -j4 Release` and makes no differences, still doesn't works.
In `ofxTCPServer::receive(int clientID)` i've tried commenting out everiting except lock and return, and the slowdown it's still here, so maybe the problem is here:
`return getClient(clientID).receive();` [https://github.com/openframeworks/openFrameworks/blob/565f9635bdd494a37ee8fb6dd2fa59ae5ae5edcd/addons/ofxNetwork/src/ofxTCPServer.cpp#L147](url)

**Comment by crx091081gb** [comment]
I have the same problem in of 10.1 on stretch on the pi3.

**Comment by crx091081gb** [comment]
I can get round the slow down by running ofTCPServer in its own thread and using ofThreadChannel to communicate back 

https://github.com/crx091081gb/networkTcpServerThreadChannelExample
